src/inmanta/deploy/persistence.py

```diff
@@ -253,7 +281,8 @@ def error_and_log(message: str, **context: Any) -> None:
                     last_deployed_version=resource_id_parsed.version,
                     last_deployed_attribute_hash=resource.attribute_hash,
                     last_non_deploying_status=const.NonDeployingResourceState(status),
-                    **extra_fields,
+                    deployment_result=deployment_result,
```

Comment:
Do I see correctly that the current implementation only writes deployment result, not compliance status or blocked status?

Reply:
I've been working on the code a bit to see where we are and I resolved this on a branch https://github.com/inmanta/inmanta-core/pull/8479/files